research-synthesis-requesters.md

```diff
@@ -0,0 +1,70 @@
+ 
+We’re especially interested in understanding how this can be done in a way that improves the system as a whole.
+ 
+### This document is NOT:
+* Finished or complete
+* A list of everything we plan to fix or address
+* Quantitative, statistically significant, or generalizable
+
+### This document IS:
+* Initial research that is broader than what we plan to work on
+* Imperfect and incomplete: please let us know what we’re missing!
+* Qualitative: this research is used to find patterns to further examine and test
+ 
+**We are sharing our initial research because we know it is imperfect and would greatly appreciate your feedback. Please [open an issue](https://github.com/18F/foia-recommendations/issues/new) in GitHub or email us at [National.FOIAPortal@usdoj.gov](mailto:National.FOIAPortal@usdoj.gov), and keep our [contributing policy](https://github.com/18F/foia-recommendations/blob/master/CONTRIBUTING.md) in mind.**
+
+### What we were researching
+
+The goal of our research is to inform hypotheses that address the following:
+* How might we create a platform that improves the FOIA requesting experience?
+* How might we make it easier for agencies to respond to requests?
+* How might various FOIA requesters benefit from a national FOIA platform?
+* How might agency processes be improved or enhanced by a national FOIA platform?
+
+### Requesters this document covers
+* Academia
+* Journalism and advocacy
+* Open gov advocacy
+* Attorneys (advocates for users who need their records)
+* Business
+
+### Requesters this document doesn’t cover
+* Agency stakeholders
+* Agency developers and IT staff
+* First-time requesters
+
+### What we learned
+
+#### Pain points
+Requesters who we spoke with often distrust the FOIA process and struggle with a lack of insight and understanding into what is happening from the time of submission to the receipt of a response from the agency. 
+ 
+Requesters have little trust that their request won’t fall into a “black hole” unless they closely track and nudge it, that it’s being handled as quickly as possible, that searches are conducted exhaustively, and that redactions are made in good faith.
+ 
+For these requesters, these concerns are significantly greater than the challenges related to submitting requests.
+
+##### High
+* Requesters have little insight into the process after submission
+* The process takes a long time
+* Requesters want access to individuals handling their request so they can check on status of the request or attempt to get their response more quickly
+* Requesters report that time estimates are rarely accurate and requesters don’t place any value or meaning in them.
+* Lack of trust in redactions. Perception of incomplete disclosure.
```

Comment:
Punctuation on these bullets is inconsistent, we should add a period to all of them and make sure they are complete sentence(s).